src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/generate_pipeline_inputs.py

```diff
@@ -76,46 +76,69 @@ def fetch_required_ssm_params(regions):
         parameter_store = ParameterStore(region, boto3)
         output[region] = {
             "s3": parameter_store.fetch_parameter(
-                f'/cross_region/s3_regional_bucket/{region}',
+                f"/cross_region/s3_regional_bucket/{region}",
             ),
             "kms": parameter_store.fetch_parameter(
-                f'/cross_region/kms_arn/{region}',
+                f"/cross_region/kms_arn/{region}",
             ),
         }
         if region == DEPLOYMENT_ACCOUNT_REGION:
-            output[region]["modules"] = parameter_store.fetch_parameter('deployment_account_bucket')
-            output['default_scm_branch'] = parameter_store.fetch_parameter('default_scm_branch')
+            output[region]["modules"] = parameter_store.fetch_parameter(
+                "deployment_account_bucket"
+            )
+            output["default_scm_branch"] = parameter_store.fetch_parameter(
+                "default_scm_branch"
+            )
     return output
 
 
-def worker_thread(p, organizations, auto_create_repositories, deployment_map, parameter_store):
-    LOGGER.debug("Worker Thread started for %s", p.get('name'))
+def worker_thread(
+    p, organizations, auto_create_repositories, deployment_map, parameter_store
+):
+    LOGGER.debug("Worker Thread started for %s", p.get("name"))
     pipeline = Pipeline(p)
-    if auto_create_repositories == 'enabled':
-        code_account_id = p.get('default_providers', {}).get('source', {}).get('properties', {}).get('account_id', {})
-        has_custom_repo = p.get('default_providers', {}).get('source', {}).get('properties', {}).get('repository', {})
-        if auto_create_repositories and code_account_id and str(code_account_id).isdigit() and not has_custom_repo:
-            repo = Repo(code_account_id, p.get('name'), p.get('description'))
+    if auto_create_repositories == "enabled":
+        code_account_id = (
+            p.get("default_providers", {})
+            .get("source", {})
```

Comment:
I wasn't aware you could do this, nice.

Reply:
FYI: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#call-chains